holoviews/element/path.py

```diff
@@ -38,7 +38,7 @@ class Path(Element2D):
 
     def __init__(self, data, **params):
         if isinstance(data, tuple):
-            x, y = data
+            x, y = map(np.asarray, *data)
```

Comment:
This should just be ``map(np.asarray, data)``, could you fix it, then I'll merge asap.